README.md

```diff
@@ -26,7 +26,7 @@ The OpenTrials API service.
 2. Run `npm install`;
 3. Run `npm run migrate`;
 4. (Optional) If you want, you can add some seed data using `npm run seed`;
-5. Run `npm run reindex`;
+5. (If you ran 4) Run `npm run reindex`;
```

Comment:
yeah I was just following the exact to-do described on the readme, so in this process the database was completely clear, causing an error when runnning the reindex, but if you consider that users can do any dumping processes between on any point of the setup, I agree with you and would mantain as it is